app/tools/android/make_apk.py

```diff
@@ -142,12 +140,36 @@ def FindExtensionJars(root_path):
         extension_jars.append(extension_jar)
   return extension_jars
 
+# try to parse out suitable app_versionCodeBase based on app_version
+# for example, app_version "3.1.0" will generate app_versionCodeBase "310"
```

Comment:
This limited version conversion logic sound reasonable to me. It should help most of cases.